### PR TITLE
Add cloud get started content

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/cloud-get-started.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Install the ReSolve CLI
+
+Type the following command to install the reSolve cloud CLI to your application.
+
+```bash
+yarn global add resolve-cloud
+```
+
+After this, use the CLI to authenticate to the reSolve Cloud Platform:
```
Comment (arminal) [inline docs/cloud-get-started.md]:
```suggestion
The next step is to use the CLI to authenticate to the reSolve Cloud Platform:
```

File: docs/cloud-get-started.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+resolve-cloud login
+```
+
+The CLI will ask you for you account credential and establish an authentication session.
```
Comment (arminal) [inline docs/cloud-get-started.md]:
```suggestion
The CLI will ask you for you account credentials and establish an authentication session.
```

File: docs/cloud-get-started.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Create a ReSolve Application
+
+Create a reSolve application. For this tutorial, you can use one of examples that come with the reSolve framework.
```
Comment (arminal) [inline docs/cloud-get-started.md]: What do you mean by "for this tutorial"? what tutorial? is the user supposed to create a tutorial?

Comment (EugeniyBurmistrov) [reply docs/cloud-get-started.md]: I mean, "To go through this tutorial" (the tutorial that we are editing)

File: docs/cloud-get-started.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Install the ReSolve CLI
+
+Type the following command to install the reSolve cloud CLI to your application.
+
+```bash
+yarn global add resolve-cloud
+```
+
+The next step is to use the CLI to authenticate to the reSolve Cloud Platform:
```
Comment (arminal) [inline docs/cloud-get-started.md]: to authenticate WHAT to the resolve cloud platform?

File: docs/cloud-get-started.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+```bash
+resolve-cloud login
+```
+
+The CLI will ask you for you account credentials and establish an authentication session.
+
+## Create a ReSolve Application
+
+Create a reSolve application. For this tutorial, you can use one of examples that come with the reSolve framework.
```
Comment (arminal) [inline docs/cloud-get-started.md]:
```suggestion
Create a reSolve application. For this tutorial, you can use one of the examples that come with the reSolve framework.
```

File: docs/cloud-get-started.md (hunk 4 above)
Comment (EugeniyBurmistrov) [reply docs/cloud-get-started.md]: I think, I choose a wrong word. Changed to "to log in to the reSolve Cloud Platform"

**Pull request merged**